meta/Decays.m

```diff
@@ -928,6 +928,14 @@ would violate Ward identity (as at the level of dim 4
                  "case 1: {\n" <>
                  TextFormatting`IndentText[
                     "auto dm = std::make_unique<" <> FlexibleSUSY`FSModelName <> "_mass_eigenstates_decoupling_scheme>(model.get_input());\n" <>
+                    If[
+                       MemberQ[
+                          DeleteCases[{TreeMasses`GetHiggsBoson[], TreeMasses`GetPseudoscalarHiggsBoson[], TreeMasses`GetChargedHiggsBoson[] /. Susyno`LieGroups`conj->Identity}, Null],
+                          particle /. Susyno`LieGroups`conj->Identity
+                       ],
+                       "dm->set_use_pole_higgs_mixings(static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::use_pole_higgs_mixings)));\n",
```

Comment:
Personally I'd like to write `x != 0.0` instead of `static_cast<bool>(x)`, because I find it more readable.

Reply:
I could omit the cast entirely and just rely on auto conversion. Would that be ok?

Reply:
Done in 58648c9e1e55c629742b75fd3a197eb4debc9c15.